invoicing_context.md

```diff
@@ -316,6 +316,7 @@ You may want to set the `source_app_id` if the invoice origin does not correspon
 | &nbsp;&nbsp;&nbsp;&nbsp;`maintenanceLogText`            | `string`  | optional uses bookingText if empty                                                                                                                              |
 | &nbsp;&nbsp;&nbsp;&nbsp;`developmentCode`               | `string`  | optional development code retrieved from a GraphQL query (codeTable developments); sometimes required see (5)                                                   |
 | &nbsp;&nbsp;&nbsp;&nbsp;`developmentAccountingPlanCode` | `string`  | optional development accounting plan code retrieved from a GraphQL query (codeTable developmentAccountingPlans); sometimes required see (5)                     |
+| &nbsp;&nbsp;&nbsp;&nbsp;`individualItemBalancingReference` | `string`  | optional reference used to individually reconcile the line item (Einzelpostensaldierend Referenz/EPSID)                                                      |
```

Comment:
Not sure if it's just a rendering issue, but shouldn't the cell borders of the first column be aligned?

Reply:
oops - I'll fix that